src/main/resources/assets/jadexic2c/lang/uk_ua.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config.jade.plugin_ic2.wrenchable_info": "Інформація від ключа",
+  "config.jade.plugin_ic2.eu_storage_info": "Інформація про енергосховища",
+  "config.jade.plugin_ic2.eu_reader": "Інформація від EU-метра",
+  "config.jade.plugin_ic2.thermometer": "Інформація від термометра",
+  "config.jade.plugin_ic2.crop": "Інформація від бібліотеки культур",
```

Comment:
This is a bit misleading. The info is provided by Crop Analyzer, not Crop Library. 
In the previous iteration of IC2 (as of 1.12.2) `Crop Analyzer` was translated into Ukranian as `Агро-аналізатор`.
Could you change that for better understanding?

Reply:
I really confused the word. 
In translation IC2, I translated `crop_analyzer_block` as `Аналізатор врожаю`.
Do you think it would be better to do it like `Агроаналізатор` if it's essentially the same word?

Reply:
In this case `Агроаналізатор` would be better 👍

Reply:
I asked my friends which option they would choose, most of them like `Агроаналізатор`
I will write the full version of this word `Агрономний аналізатор` [_Agronomic Analyzer_] so that there are no problems with understanding, and it is a little more official and complete

I'll also reduce `Інформація від термометра` to `Температура`[_Temperature_]
I also want to ask, I haven't seen these features in mods yet, so I wonder if I'll need to put `:` after the word?

Reply:
> I'll also reduce Інформація від термометра to Температура[Temperature]

The thing is that the `Thermometer` tool provides not only temperature info, but also brewing info, so keeping it as it is right now, is also acceptable.
> I also want to ask, I haven't seen these features in mods yet, so I wonder if I'll need to put : after the word?

What do you mean?
Can you elaborate on this?

Reply:
I've never done any brewing, so I haven't seen it, 
I was initially interested in the mod because of the ability to see the charge level similar to the Thermal series
![image](https://github.com/CrossVas/JadeAddonsIC2Classic/assets/62100293/d9e429b6-59e7-471d-8b84-9b93ad4c09d7)
![image](https://github.com/CrossVas/JadeAddonsIC2Classic/assets/62100293/c4f2e0ba-a615-42f7-91b3-661a087b0c23)

Reply:
In order to see additional info you have to have `EU-Reader`, `Thermometer` or `Crop Analyzer` in your hotbar ot integrated into your helmet.

Reply:
By the way, it makes sense
I'm already used to the default display of indicators